server/neptune/gateway/requirement/aggregate.go

```diff
@@ -90,3 +90,27 @@ func (a *Aggregate) Check(ctx context.Context, criteria Criteria) error {
 	}
 	return nil
 }
```

Comment:
Can you change Aggregate to DeployAggregate as well?

Reply:
yep, changed!